CONTRIBUTING.md

```diff
@@ -34,6 +40,16 @@
   }
 ```
 
+* Use single quotes in HTML/JSX
+
+```js
+function msgTextToEl(text) {
+  return <div class='Message-content'>
+    {text}
+  </div>
```

Comment:
JSX here because then I don't have to explain the use of `choo/html`. Also, we're probably going to switch to Preact sometime soon.